algebird-core/src/main/scala/com/twitter/algebird/Eventually.scala

```diff
@@ -130,7 +130,7 @@ class EventuallyMonoid[E, O](convert: O => E)(mustConvert: O => Boolean)(implici
 ) extends EventuallySemigroup[E, O](convert)(mustConvert)
     with Monoid[Either[E, O]] {
 
-  override def zero = Right(Monoid.zero[O])
+  override def zero = Right(Monoid.zero[O]) // scalafix:ok
```

Comment:
some of these need to be ignored ... inferred type is `Right` and it will break bin-compat

Reply:
wait, why not? why can't we write the type inferred? has to be either `Right[E, O]` or `Either[E, O]`. Both are fine types for zero if you ask me.